packages/math/src/conversion.ts

```diff
@@ -0,0 +1,7 @@
+export function toDegrees(radians: number): number {
+    return radians * (180 / Math.PI)
+}
+
```

Comment:
Pref: It would be cool to have some in code documentation, maybe use the JSdoc strategy implemented by Tijs in @bothrs/translations

Reply:
Just asking;

Do we need JSDoc style documentation if the Typescript .d.ts files provide information about the method and its parameters? 🤔

Reply:
It would be nice to have some info on what the function does exactly, especially since these are fancy math functions.

Reply:
I'll provide a wikipedia link :trollface: